tests/shared/src/test/scala/cats/tests/OpSuite.scala

```diff
@@ -44,13 +44,14 @@ class OpSuite extends CatsSuite {
   }
 
   {
-    implicit val catsDataDecidableForOp = Op.catsDataDecidableForOp[Function1, Int]
+    implicit val catsDataDecidableForOp: Decidable[Op[Function1, Int, *]] = Op.catsDataDecidableForOp[Function1, Int]
     checkAll("Op[Function1, Monoid, ?]", DecidableTests[Op[Function1, Int, *]].decidable[MiniInt, MiniInt, MiniInt])
     checkAll("Decidable[Op[Function1, Monoid, ?]]", SerializableTests.serializable(Decidable[Op[Function1, Int, *]]))
   }
 
   {
-    implicit val catsDataContravariantMonoidalForOp = Op.catsDataContravariantMonoidalForOp[Function1, Int]
+    implicit val catsDataContravariantMonoidalForOp: ContravariantMonoidal[Op[Function1, Int, *]] =
+      Op.catsDataContravariantMonoidalForOp[Function1, Int]
```

Comment:
If you'll forgive me, just taking a moment to inform about cross-platform Scala for any wider audience :) nothing personal!

The commit comment says:

> Fix JS linking in tests 

Actually, you are fixing vanilla Scala 3 compiler errors here (the need for explicit types on implicits). In CI sometimes the JS job happens to fail first, but that doesn't mean that the problem is specfiic to JS :)

In general, the compiler frontends for Scala JVM, JS, and Native are the same—it's only the backends that produce different output (bytecode vs javascript vs native binaries). So compile-time quirks like these related to Scala features (types, implicits, etc.) are usually general Scala things and not platform specific.

Reply:
I appreciate this a lot actually! I don't have opportunities to use Scala 3 or Scala JS so I did not know the actual ground of this.